server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/server.ftl

```diff
@@ -141,7 +141,8 @@
             <thead>
               <tr>
                 <th>Table&nbsp;</th>
-                <th>Tablet&nbsp;</th>
+                <th title="Run 'getsplits -v' in the Accumulo Shell to associate the encoded tablets with their actual splits.">Tablet&nbsp;
+                  <img height="10em" width="10em" alt="information" src="/resources/images/circled-information-icon.png"></th>
```

Comment:
The non-breaking space is moot if it is followed by a breaking space (newline, indentation). 

Also, the image tag isn't proper XML. I know HTML5 is pretty forgiving, but it's nice to have the tags be balanced with either a closing tag element or the terminal `/>` at the end of the start tag.

```suggestion
                <th title="Run 'getsplits -v' in the Accumulo Shell to associate the encoded tablets with their actual splits.">Tablet&nbsp;<img height="10em" width="10em" alt="information" src="/resources/images/circled-information-icon.png" /></th>
```

I'm more concerned, however, that the new image file added to the PR seems to be showing up as an "empty file". Not sure what's going on with that. Bug in GitHub? Did you add an empty file? There might be an existing bootstrap icon that can be used, that is already in the monitor's resources.

Reply:
Maybe `.glyphicon-info-sign` will work?

```suggestion
                <th title="Run 'getsplits -v' in the Accumulo Shell to associate the encoded tablets with their actual splits.">Tablet&nbsp;<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>&nbsp;</th>
```

Reply:
> The non-breaking space is moot if it is followed by a breaking space (newline, indentation).

I thought the template will ignore whitespace/format when generating the page (hence our use of the `&nbsp;` encodings)? 

> I'm more concerned, however, that the new image file added to the PR seems to be showing up as an "empty file". Not sure what's going on with that. Bug in GitHub? Did you add an empty file? There might be an existing bootstrap icon that can be used, that is already in the monitor's resources.

The file I committed is the image and not empty so I don't know what is going on with github.
<pre>
08:13:02 (tablet-details-monitor) ~/workspace/accumulo$ file server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/images/circled-information-icon.png
server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/images/circled-information-icon.png: PNG image data, 128 x 128, 8-bit/color RGBA, non-interlaced
08:13:07 (tablet-details-monitor) ~/workspace/accumulo$ du -sh server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/images/circled-information-icon.png
16K	server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/images/circled-information-icon.png
</pre>

Reply:
That is a possibility. I would have to track down where I got it from. I will try the bootstrap icon.